typescript/sdk/src/metadata/ChainMetadataManager.ts

```diff
@@ -286,4 +286,36 @@ export class ChainMetadataManager<MetaExt = {}> {
     }
     return new ChainMetadataManager(newMetadata);
   }
+
+  /**
```

Comment:
Pulled this up from MultiProvider to the parent class here so MultiProtocolProvider can use it as well